diff --git a/setup.2.py b/setup.2.py
index bb2e4aa..cd11d91 100644
--- setup.2.py
+++ setup.2.py
@@ -1,9 +1,11 @@
 # -*- coding: utf-8 -*-
+import os
 from distutils.core import setup, Extension
 
 module = Extension('aspell',
 	libraries = ['aspell'],
-	library_dirs = ['/usr/local/lib/'],
+	library_dirs = [os.path.join(os.getenv('PREFIX'), 'lib')],
+	include_dirs = [os.path.join(os.getenv('PREFIX'), 'include')],
 	sources = ['aspell.2.c']
 )
 
